src/website/app/demos/Link/bestPractices.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Link from '../../../../Link';
+
+export default [
+  {
+    type: 'do',
+    title: 'use Links to navigate',
+    example: (
+      <p>
+        <Link href="https://mineral-ui.com">Lorem ipsum</Link> dolor sit amet,
+        consectetur adipiscing elit. Donec mattis pretium massa. Aliquam erat
+        volutpat. Nulla facilisi. Donec vulputate interdum sollicitudin.
+      </p>
+    ),
+    description:
+      'Links should be used inline to navigate to other content or pages.'
+  },
+  {
+    type: 'dont',
+    title: 'use a link to perform a page action',
+    example: <Link>Scroll to bottom</Link>,
+    description:
+      'Links should be used for navigation. To call out an action, use a [Button](./button).'
```

Comment:
Remove the first sentence, as you just said it in the "do" practice?

"To _represent_ an action..."? "call out" connotes CTAs.